.github/workflows/mobile-curriculum-e2e.yml

```diff
@@ -138,7 +138,7 @@ jobs:
           force-avd-creation: false
           emulator-options: -no-snapshot-save -no-window -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
           disable-animations: true
-          script: cd mobile/mobile-app && if adb shell pm list packages | grep -q org.freecodecamp; then adb uninstall org.freecodecamp; fi && dart integration_test_runner.dart --curriculum ${RUNNER_DEBUG:+--debug}
```

Comment:
This was done so debug flag is set when you enable "verbose logging" when you re-run an action as seen in the below image

<img width="624" height="204" alt="Image" src="https://github.com/user-attachments/assets/ceec4d52-1e3e-4a83-84eb-c6fe2b5c2be1" />